plugins/services/services_inspector.rb

```diff
@@ -107,7 +107,8 @@ def parse_redhat_chkconfig
       stdout: :capture
     )
 
-    services = output.lines.map do |line|
+    services = output.lines.select{|line| line =~ /^\w.*0:(on|off)\s*1:(on|off)\s*2:(on|off)\s*3:(on|off)\s*4:(on|off)\s*5:(on|off)\s*6:(on|off).*$/}.map do |line|
```

Comment:
Line is too long. [163/100]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.